python/lsst/daf/butler/core/dataUnit.py

```diff
@@ -260,6 +272,36 @@ def items(self):
         for dataUnitName in self._dataUnitNames:
             yield (dataUnitName, self[dataUnitName])
 
+    def getRegionTable(self, *dataUnitNames):
+        """Return the region table that holds regions for the given combination
+        of DataUnits.
+        """
+        if len(dataUnitNames) == 1:
+            return self[dataUnitNames[0]].table
```

Comment:
`self._dataUnits[dataUnitNames[0]].table`?

Reply:
Seems better to have private methods use the public API unless there's a good reason not to.

Reply:
I missed `__getitem__`, sorry for the noise.